test/Microsoft.VisualStudio.Editor.Razor.Test/DefaultRazorDocumentManagerTest.cs

```diff
@@ -31,8 +31,6 @@ public class DefaultRazorDocumentManagerTest : ForegroundDispatcherTestBase
 
         private ImportDocumentManager ImportDocumentManager => Mock.Of<ImportDocumentManager>();
 
-        private Workspace Workspace => new AdhocWorkspace();
```

Comment:
You don't really need to rewrite all of this, just use your new class to create the workspace